test/driver_test.rb

```diff
@@ -128,9 +132,101 @@
 
       expect(@driver.average_rating).must_be_close_to (5.0 + 1.0) / 2.0, 0.01
     end
+
+    it "average rating works with in-progress trips" do
+
+      @driver = RideShare::Driver.new(
```

Comment:
You should avoid reassigning instance variables in tests, it can cause some confusing bugs.

Instead create a local variable and reference it below:
```suggestion
      driver = RideShare::Driver.new(
```